diff --git a/node_modules/@streamq/player/dist/index.js b/node_modules/@streamq/player/dist/index.js
index 310dea6..5309eaa 100644
--- a/node_modules/@streamq/player/dist/index.js
+++ b/node_modules/@streamq/player/dist/index.js
@@ -10859,12 +10859,11 @@ var hp = class {
   }
 };
 function mp(e) {
-  return () => new hp(e);
+  return (_displayRoot) => new hp(e);
 }
 function gp(e) {
   return Object.freeze({
-    textDisplayFactory: mp(e),
-    streaming: Object.freeze({ alwaysStreamText: !0 })
+    textDisplayFactory: mp(e)
   });
 }
 function vp(e, t) {
